sync15-adapter/Cargo.toml

```diff
@@ -22,3 +22,7 @@ failure_derive = "0.1.1"
 [dev-dependencies]
 env_logger = "0.5"
 prettytable-rs = "0.6"
+fxa-client = { path = "../fxa-rust-client" }
+
+[[example]]
+name = "sync-pass"
```

Comment:
What does defining the name change? (`cargo run --example sync-pass` should have already worked -- I'm not opposed to explicitly listing it here even if there's no concrete change though)

Reply:
> (cargo run --example sync-pass should have already worked)

I checked, and indeed it does work without listing it explicitly.  This was probably just me working my way through the cargo docs leaning how to run examples and cargo-culting things into the config file, not realizing that it wasn't necessary.